pydoctor/templates/apidocs.css

```diff
@@ -195,30 +202,90 @@ tr.private {
     margin-bottom: 6px;
 }
 
+/* This is magic to show the transparent index link */
+a.navbar-brand {
+    /* background-color:#f3d3db; */
+    border:solid;
+    margin-right: 0.3em;
+    border-width: 1.5px;
+    border-color: #c7254e;
+    background-color:#f9f2f4;
+    border-radius: 4px;
+}
+
+/* This is for the main top navagation bar */
+.page-header > h1 > code {
+    background-color:#f9f2f4;
```

Comment:
Maybe we should attach the color to the `<h1>` tag rather than the `<code>` tag. Then we could later use the `package`/`module`/`class` styles that are currently applied to `<tr>` also on headers.

Reply:
With `.page-header > h1 > * {` instead of `.page-header > h1 > code {` it works

Reply:
Hmm, not sure that's an improvement. I was thinking of using just `.page-header > h1 {`, but it seems quite a lot of CSS needs to change for that to work.

Also the `<h1>` spans the entire body width instead of just the title, so it would look differently.

Let's leave it as- is, for now.